frontend/src/store/ux.js

```diff
@@ -37,7 +37,7 @@ const getters = {
         return state.tours.education
     },
     mainNavContexts: function (state, getters, rootState, rootGetters) {
```

Comment:
should these be `store` and `rootStore`?

Given that the input to `usePermissions` is `store`, but you're passing a variable called `rootState`?

Reply:
It would depend on the context where they are used, I used the naming vuex describes them in their [api](https://vuex.vuejs.org/api/#getters). It would be better if i'd change the argument name in the composable to rootState to match it